packages/website/src/styles.css

```diff
@@ -1,58 +1,64 @@
-@import url('https://fonts.googleapis.com/css2?family=Alegreya:ital,wght@0,400;0,500;0,600;0,700;0,800;0,900;1,400;1,500;1,600;1,700;1,800;1,900&display=block');
-@import url('https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,400;0,500;0,600;0,700;0,800;0,900;1,400;1,500;1,600;1,700;1,800;1,900&display=block');
-@import url('https://fonts.googleapis.com/css2?family=Inter&display=block');
-@import url('https://fonts.googleapis.com/css2?family=Fira+Code&display=block');
-@tailwind base;
-:root {
-  @apply bg-black;
+@import url('https://fonts.googleapis.com/css2?family=Alegreya:ital,wght@0,400;0,500;0,600;0,700;0,800;0,900;1,400;1,500;1,600;1,700;1,800;1,900&display=block')
```

Comment:
Missing semicolons?

Reply:
hmm it's supposed to look like this: https://tailwindcss.com/docs/preflight#overview

prettier is reformatting it to this: (line added)

```css
@import url('https://fonts.googleapis.com/css2?family=Inter&display=block')
layer(base);
```

maybe we need a change in the prettier config or add a plugin? I'll look

Reply:
So I don't think the layer on a newline is technically a syntax error (the styles still work when it is like this)

but we can add `/* prettier-ignore */` before each of these `@import` calls if you'd prefer to see them each on their own line. The rule that it's hitting is the line width, so it is wrapping the line. You can double check this by running prettier on `@import url('https://fonts') layer(base);` (it will not line wrap)

Reply:
I was just curious about the diff. If it's intentional, it's fine. I'm not suggesting to ignore prettier.

Reply:
Ok then, I'll leave it 👍